qiskit/transpiler/passes/scheduling/alap.py

```diff
@@ -34,15 +33,17 @@ class ALAPSchedule(BaseSchedulerTransform):
         removed in a future release.
     """
 
+    @deprecate_func(
+        # TODO: This grammar doesn't make sense
```

Comment:
How should I fix this?

Reply:
it should be "[use ALAPScheduleAnalysis], which is an analysis pass that ..."